build/generated.bzl

```diff
@@ -0,0 +1,149 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The go_prefix for this repository, needed to map relative package paths into
+# a full-resolved go package.
+go_prefix = "k8s.io/kubernetes"
+
+# Pretty-prints the provided dict -> dict -> dict as a dict -> dict -> list,
+# appending lines into output.
+def _format_output_dict(id, tags, output):
+    output.append("%s = {" % id)
+
+    for tag in tags.keys():
+        d = tags[tag]
+        output.append("%s%s: {" % (4 * " ", repr(tag)))
```

Comment:
Can we indent with 2 spaces instead of 4?

Reply:
skylark follows PEP 8, i.e. 4-space indentation.